### PR TITLE
Removed long_poll option

### PR DESCRIPTION
**Pull request by Neamar**

### CODE REVIEW

File: guides/using/hydrater.md [hunk 1]
```diff
@@ -20,26 +20,60 @@ Once previous tasks are finished, the hydrater will download the file from `file
 Hydration will then occur. Once completed, the endpoint you specified as `callback` will be pinged with a `PATCH` verb and a json payload.
 
 #### Testing
-For cases where you want to test the results and don't want to ping another adress with the result, you can then use the `long_poll` option. This is not for production use, and only for testing purposes (anyway, in production, if the hydrater is busy, your request will be dropped). The `long_poll` option returns the result with the request (instead of instantly returning 202).
+For cases where you want to test the results and don't have another adress to ping, you can use the `http://echo.anyfetch.com` server. This server will simply "echo" back the received requests. For instance, you can call the OCR with this:
```
Comment (Amoki) [inline guides/using/hydrater.md]: address

File: guides/using/hydrater.md [hunk 2]
```diff
@@ -54,7 +88,7 @@ $ curl --header "Content-Type:application/json" --data '{"file_path":"https://ra
 In this particular case, nothing was changed, but some hydraters can use the initial data to compute and improve their results.
 
 #### Hydrater status
-Hydrater queue can get quite long sometimes. You may want to check the status of a hydrater using `/status` endpoint:
+Hydrater queue can get quite long sometimes. You can check the status of a hydrater using `/status` endpoint:
```
Comment (Amoki) [inline guides/using/hydrater.md:91]: already said some lines upper

Comment (Neamar) [inline guides/using/hydrater.md:91]: i know. I think it's useful to state it twice

**Comment by rricard** [comment]
Looks good to me !

**Pull request merged**